src/Server.php

```diff
@@ -90,6 +106,7 @@
 class Server extends EventEmitter
 {
     private $callback;
+    private $serverAddress;
```

Comment:
Should probably use `$conn->getLocalAddress()` instead in case this is a "listen all" address?